internal/cluster/cmd/clusterd/main.go

```diff
@@ -44,7 +44,7 @@ func main() {
 	flag.StringVar(&dir, "dir", "", "directory to store renter data")
 	flag.StringVar(&apiAddr, "api", ":3001", "API address")
 	flag.StringVar(&logLevel, "log", "info", "logging level")
-	flag.StringVar(&network, "network", "v1", "network to use (v1 or v2)")
+	flag.StringVar(&network, "network", "v1", "network to use (v1, v2, or transition)")
```

Comment:
```suggestion
	flag.StringVar(&network, "network", "v1", "network to use (v1, v2 or transition)")
```

Reply:
Do you not use a comma there haha? I changed it.